src/test/java/igym/entities/GymTest.java

```diff
@@ -76,4 +77,16 @@ void testNameTooLong() {
         assertFalse(violations.isEmpty());
         assertEquals("Name must be between 3 and 50 characters", violations.iterator().next().getMessage());
     }
+
+    @Test
+    @DisplayName("should set the gym status")
+    void testSetGymStatus(){
+        Gym gym = new Gym();
+        gym.setStatus(GymStatus.inactive);
+        Set<ConstraintViolation<Gym>> violations = validator.validate(gym);
+        assertFalse(violations.isEmpty());
+        assertTrue(gym.getStatus() == GymStatus.inactive);
+    }
```

Comment:
Daniel, I suggest you to also test to set an invalid status on this test.
```suggestion
    void testSetGymStatus(){
        Gym gym = new Gym();
        // Sugestion ⬇️ 
        gym.setStatus("invalid status");
        //assert invalid and continue the test
        
        gym.setStatus(GymStatus.inactive);
        Set<ConstraintViolation<Gym>> violations = validator.validate(gym);
        assertFalse(violations.isEmpty());
        assertTrue(gym.getStatus() == GymStatus.inactive);
    } 
```

Reply:
The problem with that approach is that the method setStatus cannot receive values different from those defined in the Enum class. It accepts a null value